src/pages/register-a-beacon/aircraft-communications.tsx

```diff
@@ -66,10 +71,10 @@ const TypesOfCommunication: FunctionComponent<{ form: FormJSON }> = ({
 }: {
   form: FormJSON;
 }) => (
-  <FormFieldset>
-    <FormLegend size="small">
+  <>
+    <FormHint forId="aircraft-communication-types">
```

Comment:
@SianRMadeTech This pattern of aria described by hint should also be applied in the `beacon-information` page

Reply:
changes made